kubernetes/manifests/kashira.yml

```diff
@@ -1,3 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kashira-config
+  namespace: katana
+data:
+  daemon_port: "{{.DaemonPort}}"
```

Comment:
Why is this required?

Reply:
Because percy said not to hardcode so I have set the env in yml file